pkg/controller/podgc/gc_controller.go

```diff
@@ -238,7 +240,13 @@ func (gcc *PodGCController) gcOrphaned(ctx context.Context, pods []*v1.Pod, node
 			continue
 		}
 		klog.V(2).InfoS("Found orphaned Pod assigned to the Node, deleting.", "pod", klog.KObj(pod), "node", pod.Spec.NodeName)
-		if err := gcc.deletePod(ctx, pod.Namespace, pod.Name); err != nil {
+		condition := &v1.PodCondition{
+			Type:    v1.AlphaNoCompatGuaranteeDisruptionTarget,
+			Status:  v1.ConditionTrue,
+			Reason:  "DeletionByPodGC",
+			Message: "Podgc: node no longer exists",
```

Comment:
I see `PodGC` in most of the logging messages, but I don't know which format is correct

Reply:
Indeed, PodGC is produced in this log message: https://github.com/kubernetes/kubernetes/blob/8e14585d8700d9cf7861be66943f023d11d41075/pkg/controller/podgc/gc_controller.go#L312

Let me stick to it in the message here as well.